src/app/components/Metadata/index.tsx

```diff
@@ -134,7 +148,7 @@ const MetadataContainer = ({
         content="width=device-width, initial-scale=1, minimum-scale=1"
       />
       <title>{pageTitle}</title>
-      <link rel="canonical" href={canonicalNonUkLink} />
+      <link rel="canonical" href={canonicalToUse} />
```

Comment:
Worth having a test around this to check the `href` for these different conditions?